doc/source/ray-core/examples/batch_prediction.ipynb

```diff
@@ -7,7 +7,7 @@
     "# Batch Prediction\n",
     "\n",
     "```{tip}\n",
-    "Ray provides multiple ways to implement this workload.\n",
+    "Ray provides multiple ways to implement this workload. As a general rule, for simple, naively parallel workloads like this one:\n",
     "\n",
     "* Ray Core will be the lowest level of abstraction and the most performant.\n",
```

Comment:
Lowest abstraction, but how most performant is achieved?

Reply:
Least overhead, ability to tailor to the specific workload more